### DISCUSSION TITLE
Affiliation with intuitions and work places

### DISCUSSION THREAD

**Discussion by Pranav400-source** [Tiham2001/tarp#7] [Ideas]
Since, we are going to take data from many institutions, it would be necessary for us to know a way or a medium of communication with them. Drop your ideas below about how we should attempt doing this.

**Comment by annanya1803** [comment]
I think we can mail them and ask for the data.

**Reply by Pranav400-source** [reply]
> I think we can mail them and ask for the data.

Yes we can, other option is to monitor the university website on regular basis. This process may not be manual.

**Reply by Tiham2001** [reply]
We can refer to LinkedIn also for reference

**Comment by shalinimishra31** [comment]
We can use many datasets from Kaggle about a survey conducted by stackoverflow.

**Reply by Tiham2001** [reply]
Wow that's a good idea😀

**Reply by annanya1803** [reply]
I think this will work
